### PR TITLE
Updated Fix for Invalid UMB Content

### PR DESCRIPTION
**Pull request by Pratster95**

### CODE REVIEW

File: vars/sendMetricsMsg.groovy [hunk 1]
```diff
@@ -11,7 +11,7 @@ import org.centos.contra.pipeline.Utils
  * @param metricsMap: A map containing the metrics to be sent over the bus
  * @return
  */
-
+string b = ""
```
Comment (johnbieren) [inline vars/sendMetricsMsg.groovy:14]: Why is not not inside of the function definition?

Comment (Pratster95) [reply vars/sendMetricsMsg.groovy:14]: I was trying to make it global. Since it is only used in call function, I will add it inside the function

Comment (johnbieren) [reply vars/sendMetricsMsg.groovy:14]: Yeah I don't see any reason to have it be global

File: vars/sendMetricsMsg.groovy [hunk 2]
```diff
@@ -13,6 +13,7 @@ import org.centos.contra.pipeline.Utils
  */
 
 def call(Map metricsMap) {
+    string b = ""
```
Comment (johnbieren) [inline vars/sendMetricsMsg.groovy:16]: can we just declare the string on line 30? it is only used in that if

File: vars/sendMetricsMsg.groovy [hunk 3]
```diff
@@ -26,6 +26,7 @@ def call(Map metricsMap) {
         def params = []
         if (metricsMap['service']['params']) {
             metricsMap['service']['params'].each {
+                string b = ""
                 b = "$it".replaceAll('"','')
```
Comment (johnbieren) [inline vars/sendMetricsMsg.groovy:30]: can do we `string b = "$it".replaceAll('"','')` ?

**Pull request merged**